Makefile

```diff
@@ -11,6 +11,7 @@ GOLANG_BASE	= presidio-golang-base
 GIT_TAG   = $(shell git describe --tags --always 2>/dev/null)
 VERSION   ?= ${GIT_TAG}
 PRESIDIO_LABEL := $(if $(PRESIDIO_LABEL),$(PRESIDIO_LABEL),$(VERSION))
+PRESIDIO_DEPS_LABEL := $(if $(PRESIDIO_DEPS_LABEL),$(PRESIDIO_DEPS_LABEL),'latest')	
```

Comment:
Sorry if I missed that previously. Isn't this going to be problematic if I didn't specify a value for PRESIDIO_DEPS_LABEL? I might push the latest. Am I correct?

Reply:
it means using 'latest' as the default for pulls mostly. 
the makefile allows pushing as well, but you have to run the specific command to do it.

Reply:
also, it corresponds to the previous way builds were done and was added to support the previous CIs